exporter/logzioexporter/README.md

```diff
@@ -169,3 +169,173 @@ service:
     logs:
       level: debug #activate debug mode
 ```
+
+#### Scope Name
+When using the logs exporter with logs originating from instrumentation library (i.e opentelemetry log4j2 appender), the scopeName field will be added (if the field is populated in the original log). 
+
+
+| Status                   |                       |
```

Comment:
Looks like starting from this all this text is a duplicate?

Reply:
@tamir-michaeli was this added by accident?

Reply:
@codeboten indeed, removed with latest commit.